models/api.model.js

```diff
@@ -0,0 +1,9 @@
+const endpointsObj = require('../endpoints.json');
+
+const selectEndpoints = () => {
+  console.log("In selectEndpoints() in api.model!");
+
+  return endpointsObj;
```

Comment:
as it is just requiring in data, I feel model might be overkill - but not incorrect or a big problem!